Dockerfile.in

```diff
@@ -8,3 +8,10 @@ RUN helm init -c && \
     helm plugin install https://github.com/hypnoglow/helm-s3.git && \
     rm -f /var/cache/apk/*
 RUN helm repo add infobloxcto s3://infoblox-helm-dev/charts
+FROM alpine/helm:VERSION
+COPY --from=0 /tmp /tmp/
```

Comment:
What is in `/tmp` that is worth copying?

Reply:
Oh, I see now. `/tmp` is `HELM_HOME`, so you are copying the repo cache.
https://github.com/infobloxopen/helm/blob/f338f26ac64e342db0d2f1d79ec46b583a2bbdd4/Dockerfile.in#L5

Next question is if it's really best to hydrate the cache at build time and distribute it, or if it would be better to hydrate the cache at runtime with an entrypoint (already merged). The cache will undoubtably be stale by the time the image is used, so bundling the cache provides a false sense of utility.